File: IdentityServer/trone/angularclient/Models/Feed.cs
using angularclient.DbAccess;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace angularclient.Models
{
    public class Feed : IEntity
    {
        [Key]
        public string Code { get; set; }
        public string UserName { get; set; }
        public DateTime DateTimeStamp { get; set; }
        public string OperationType { get; set; }
    }
}

File: IdentityServer/trone/angularclient/ClientApp/src/app/models/Feed.ts
export class Feed {
    code: string;
    userName: string;
    dateTimeStamp: Date;
    operationType: string;

    constructor(userName: string, dateTimeStamp: Date, operationType:string) {
        this.userName = userName;
        this.dateTimeStamp = dateTimeStamp;
        this.operationType = operationType;
    }
}
